### ISSUE TITLE
Use FluidMLPClassifier to predict vector of probabilities for classes

### ISSUE THREAD

**Issue by relativeflux** [flucoma/flucoma-sc#92]
Hi,

Is there a way (hopefully without having to modify the core lib) to get the FluidMLPClassifier to return a confidence vector when doing prediction? I mean smething like scikit MLPClassifier's [predict_proba](https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html#sklearn.neural_network.MLPClassifier.predict_proba). This is for doing multiclass prediction. In TensorFlow I would do this with a softmax output layer and train with categorical_crossentropy.

Thanks

**Comment by tremblap** [comment]
(if this is too abstract, I can code this example very quickly for you and post it on discourse as many people would like to see that I reckon)

**Comment by relativeflux** [comment]
Oh cheers (and thanks for the quick reply!). A code example would be great if you could provide it.

**Comment by tremblap** [comment]
incoming!

**Comment by tremblap** [comment]
check this: https://discourse.flucoma.org/t/using-mlpregressor-to-get-the-confidence-vector-of-mlpclassifier/1387

**Comment by relativeflux** [comment]
Much obliged, thank you. :)
